### DISCUSSION TITLE
1

### DISCUSSION THREAD

**Discussion by ghost** [doublecmd/doublecmd#1129] [Suggestions]

**Comment by Skif-off** [comment]
It's already over the edge, even absurdity must have some limits.

**Comment by timur-g** [comment]
Please close, good laugh, but that is it.

**Comment by Skif-off** [comment]
Why? Suggestions to change/modify the icon sometimes appear, this situation can also be an argument.
(But your icon example is too similar to Total Commander's icon, it's not good.)

P.S. I'm not laughing, I'm sad. Next step? Maybe disable mathematics, physics, engineering and many other areas of activity? They use [sinusoids](https://en.wikipedia.org/wiki/Sine_wave), [parabolas](https://en.wikipedia.org/wiki/Parabola) and other curves...
On the other hand, we can all be glad that some women don't have real problems.

**Comment by ags1234** [comment]
LOL, people are sometimes like that. If his female co-workers complained about about such a thing and then when explaining to them how off they were, they understand, then it is almost okay.
It they don't understand, they have real problems.
Back on topic, if someone could make or find a better icon, it would be interesting.
Maybe we could have a separate discussion on this matter containing suggestions, ideas, alternative icons etc.

**Comment by ghost** [comment]
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/9e233e1f-1f41-49ab-af91-dbaa6476e259)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/8672c5d1-04dc-49f7-ba3c-9f6dec644d84)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/8a2616f7-c2e2-4f93-960f-93f4f88c61e4)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/47f8749d-0662-4ead-9afa-6cf43929521a)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/0bc387a6-efde-4f84-a6a6-8584e4dfd95d)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/60c6d530-230a-4fa6-9cc9-3cc9237de51c)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/71a92977-5fd1-4242-ae60-c07cab228680)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/163222b5-88fb-4624-9fe3-6178456d56e5)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/ea2caee7-36bc-4f28-a8cc-5193436de50e)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/9fbf7034-f4cb-47d9-94c5-497aa1883d46)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/8d64aa71-27e5-4d59-bac0-01fe63fd3446)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/309c6c2c-dd84-4c1f-ab74-718894c4f024)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/470192b5-13b2-4e57-ab38-6b340e71e7e5)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/870070aa-cc26-4180-b3b0-6878e52c1ab7)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/dac083db-b341-45a0-8909-1ce13ae6dfd3)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/b790e576-1dc6-4609-9eab-d2e55b7bb69b)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/a6efeb6b-4f56-4cb6-ad49-7d396ba0652e)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/161a9722-e8d9-467a-8484-ecf5ede0fbab)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/acd24885-9fd8-4f6f-88d9-9ebfc3ebb6fd)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/b28860c5-2d4b-4b50-953c-888522eca255)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/c0d7c221-e672-47b6-9386-7164087bcb25)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/c3669b4c-94bd-454f-a3a9-25957b77b60f)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/e01d2622-7514-4d1d-9930-d90bb4119a11)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/70745145-c093-4ac0-acda-3b3169bfaf4f)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/b89a0423-696d-4f2e-aac7-5e453cb36be1)
![android-chrome-512x512](https://github.com/doublecmd/doublecmd/assets/122004809/454dc488-e3df-4ec3-9044-5fd0e7d3f88f)

**Comment by ghost** [comment]
_oops too big icons ?
github does not resize them....._

**Comment by j2969719** [comment]
the current logo is a reference not only to the letters dc, but also to the two panels.

_comicsans letters and blue floppy looks total cringe_

**Comment by ags1234** [comment]
I am satisfied with the current icon, but it does not hurt to keep an open mind :)

![istockphoto-880622992-612x612](https://github.com/doublecmd/doublecmd/assets/21011513/31cf8d03-7466-4155-a991-e7eb28003339)

source: https://media.istockphoto.com/id/880622992/vector/book-icon.jpg?s=612x612&w=0&k=20&c=nyF3k2cerHElXSVXqtcKNWmSIVR3VS39RoKo-iZVXGQ=

- it represents the two panels, specific to orthodox file managers
- it has a few pages, which represent the multitude of files/directories
- it is simple, elegant, a very stylish image of a book, an object that holds information organized in a certain way

**Comment by Skif-off** [comment]
I saw an icon somewhere that used square brackets (in the Porteus icon theme?), I thought: maybe just replace the cut and unfolded circle with a polygon?
For example, 14 or 16 edges/sides?
As a result we will have a familiar and recognizable icon, but less smooth lines. And, I think, less likely to get into an awkward situation :)

**Comment by alexx2000** [comment]
https://github.com/doublecmd/doublecmd/blob/master/pixmaps/mainicon/papirus/doublecmd.svg
